scripts/ci/gke.sh

```diff
@@ -157,7 +157,7 @@ create_cluster() {
     if [[ "${POD_SECURITY_POLICIES}" == "true" ]]; then
         PSP_ARG="--enable-pod-security-policy"
     fi
-    zones=$(gcloud compute zones list --filter="region=$REGION" | grep UP | cut -f1 -d' ' | shuf)
+    zones=$(gcloud compute zones list | grep "${REGION}\s*UP" | cut -f1 -d' ' | shuf)
```

Comment:
nitpick: This looks good. I would suggest going a step forward and avoid using "text" output. Such outputs are intended for human readability and can be changed. I would consider using JSON. Then we are a bit safer.

An example:
```
zones=$(gcloud compute zones list --format="json" | jq --arg REGION "$REGION" '.[] | select(.region | endswith($REGION)) | .name' --raw-output)
```

Reply:
Solution with predefined format and `awk` looks like a solid solution. 👍